packages/cli/src/commands/env/index.ts

```diff
@@ -126,6 +140,22 @@ export default async function main(client: Client) {
   const subArgs = argv._.slice(1);
   const { subcommand, args } = getSubcommand(subArgs, COMMAND_CONFIG);
   const { output, config } = client;
+
+  const environmentArg = argv['--environment']?.toLowerCase();
+
+  const target = environmentArg
+    ? parseEnvironmentTarget(environmentArg)
+    : ProjectEnvTarget.Development;
```

Comment:
Let's throw when an invalid target is provided instead of falling back to development

Reply:
This is undefined when the user passes in an invalid argument—it's only `.Development` when they don't. Will add a comment to clarify!